src/test/kotlin/net/maizegenetics/phgv2/api/ExportHaplotypeGraphTest.kt

```diff
@@ -72,6 +72,54 @@ class ExportHaplotypeGraphTest {
 
     }
 
+    @Test
+    fun testMultipleFilesHaplotypeGraphRangeSampleGamete() {
+
+        val graph = HaplotypeGraph(
+            listOf(
+                TestExtension.smallseqLineAHvcfFile,
+                TestExtension.smallseqLineBHvcfFile,
+                TestExtension.smallseqRefHvcfFile
+            )
+        )
+
+        val outputFile = "${outputHvcfDir}testMultipleFilesHaplotypeGraphRangeSampleGamete.vcf"
+
+        exportMultiSampleHVCF(graph, outputFile, TestExtension.smallseqRefFile, SymbolicAllele.RANGE_SAMPLE_GAMETE)
+
+        var checksum1 = getChecksum(TestExtension.exportGraphMultiSampleRangeSampleGamete)
```

Comment:
really minor but you might want to make it be checksumExpected and checksumObserved.  This way you can keep track of what is what very easily by looking at the code.  You have it in the log message but you might as well make it the variable name.

Same for the 2nd test.